extensions/interactions/Continue/directives/Continue.js

```diff
@@ -30,30 +30,42 @@ oppia.directive('oppiaInteractiveContinue', [
       templateUrl: UrlInterpolationService.getExtensionResourceUrl(
         '/interactions/Continue/directives/' +
         'continue_interaction_directive.html'),
-      controller: ['$scope', '$attrs', function($scope, $attrs) {
-        $scope.buttonText = HtmlEscaperService.escapedJsonToObj(
-          $attrs.buttonTextWithValue);
+      controller: [
+        '$scope', '$attrs', 'WindowDimensionsService',
+        'UrlService', 'EVENT_PROGRESS_NAV_SUBMITTED',
+        function($scope, $attrs, WindowDimensionsService,
```

Comment:
Same thing here and elsewhere:

```
        function(
            $scope, $attrs, WindowDimensionsService,
            UrlService, EVENT_PROGRESS_NAV_SUBMITTED) {
          $scope.buttonText...
```

The extra indent of 2 for the function args sets them apart from the line after that where the inner scope actually starts.